dali/test/python/test_video_pipeline.py

```diff
@@ -198,6 +200,30 @@ def test_file_list_starts_ends_videopipeline():
         yield _test_file_list_starts_videopipeline, r[0], r[1]
 
 
+def test_file_names_meta():
+    files = []
+    for cont in video_types:
+        path = os.path.join(video_containers_data_root, cont)
+        files += [path + '/' + f for f in os.listdir(path)]
+    files = sorted(files)
+    list_file = tempfile.NamedTemporaryFile(mode="w", delete=False)
+    for f in files:
+        list_file.write("{} {} {} {}\n".format(os.path.join(VIDEO_DIRECTORY, f), 0, 0, 1))
```

Comment:
Aren't `files` already abolute paths to files inside subdir of the DALI_extra? In that case the `os.path.join(VIDEO_DIRECTORY, f)` would be just `f`?

Reply:
True, fixed